src/controller.cpp

```diff
@@ -445,6 +445,16 @@ void Controller::handshakeTimeout() {
   activateInternal();
 }
 
+void Controller::setCooldownForAllServersInACity(const QString& countryCode,
+                                                 const QString& cityCode) {
+  logger.debug() << "Set cooldown for all servers in a city";
```

Comment:
imho, we should have a `Q_ASSERT(!Constants::inProduction())` - so everyone knows this is for debugging / testing

Reply:
That is a good point! Unfortunately we’ll need to be able to use the method in production as well so QA can test the feature — I’ll add a comment for now if that is OK.